src/views/router_views/accounts_submitted/accounts_submitted.njk

```diff
@@ -0,0 +1,54 @@
+{% extends "layouts/default.njk" %}
+
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{% block main_content %}
+    {{ govukPanel({
+        titleText: "Accounts submitted",
+        html: "Your reference number<br><strong>" + transactionId + "</strong>"
+    }) }}
+
+    {{ govukTable({
+        firstCellIsHeader: false,
+        rows: [
+                [
+                    {
+                        text: "Company number" 
+                    },
+                    {
+                        text: companyProfile.companyNumber
+                    } 
+                ],
+                [
+                    {
+                        text: "Company name" 
+                    },
+                    {
+                        text: companyProfile.companyName
+                    } 
+                ],
+                [
+                    {
+                        text: "Payment received" 
+                    },
+                    {
+                        text: "£" + payment
+                    } 
+                ]
+            ]   
```

Comment:
There isn't always a payment, so if not, we don't want to see this column, you may have to set a boolean property to check this